helpers/logger.py

```diff
@@ -1,9 +1,13 @@
 import logging
 
+
 def setup_logger(logger_name, log_level="warning"):
     # type: (str, str) -> logging.Logger
-    # todo : read level from config here
-    logger_level_attr = getattr(logging, log_level.upper())
+    try:
+        logger_level_attr = getattr(logging, log_level.upper())
+    except AttributeError:
+        logger_level_attr = logging.WARNING
```

Comment:
I did comment on this commit earlier already =) Merging it, but will improve it myself now.